packages/rest/src/keys.ts

```diff
@@ -156,6 +156,14 @@ export namespace RestBindings {
    * Binding key for setting and injecting an OpenAPI spec
    */
   export const API_SPEC = BindingKey.create<OpenApiSpec>('rest.apiSpec');
+
+  /**
+   * Binding key for setting and injecting an OpenAPI operation spec
+   */
+  export const OPERATION__SPEC = BindingKey.create<OperationObject>(
```

Comment:
Why two underscores?

```suggestion
  export const OPERATION_SPEC = BindingKey.create<OperationObject>(
```

I also think the name should communicate the fact that this is the operation spec of the endpoint being currently invoked. For example:

```suggestion
  export const CURRENT_OPERATION_SPEC = BindingKey.create<OperationObject>(
```